src/macho.rs

```diff
@@ -282,6 +283,33 @@ pub const VM_PROT_WRITE: u32 = 0x02;
 /// execute permission
 pub const VM_PROT_EXECUTE: u32 = 0x04;
 
+#[repr(u8)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum PtrauthKey {
```

Comment:
This isn't part of the file format that I can see from the usage in this PR. I think it should be moved into `read::macho`, and probably given a better name. I assume this is specific to arm, so it would be good to have that in the name.